packages/yourloops/components/statistics/patient-statistics.tsx

```diff
@@ -73,8 +86,14 @@ export const PatientStatistics: FunctionComponent<PropsWithChildren<PatientStati
           <Divider sx={{ marginBlock: theme.spacing(1), backgroundColor: theme.palette.grey[600] }} />
           <SensorUsageStat sensorUsageData={sensorUsageData} />
           <Divider sx={{ marginBlock: theme.spacing(1), backgroundColor: theme.palette.grey[600] }} />
+          {insufficientData &&
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** We only display this widget when we have insufficientData? Do we do something particular when we have enough data?
I don't know this widget or what it is, can you explain it to me please?‏